aimon/extensions/react.py

```diff
@@ -0,0 +1,187 @@
+import os
+from aimon import Client
+from typing import Optional
+from dataclasses import dataclass
+
+import random
+import string
+
+def generate_random_string(length):
+  """Generates a random string of letters and digits."""
+  characters = string.ascii_letters + string.digits
+  return ''.join(random.choice(characters) for i in range(length))
+
+@dataclass
+class ReactConfig:
+    publish: bool
+    max_attempts: int
+    hallucination_threshold: float
+    framework: Optional[str] = None
```

Comment:
make this an enum instead of a string so that only a select few values are valid values https://docs.python.org/3/library/enum.html Right now, it will only have `LLAMAINDEX` since we only support  LLamaIndex